src/dune_rules/fetch_rules.ml

```diff
@@ -0,0 +1,275 @@
+open Import
+open Memo.O
+module Gen_rules = Build_config.Gen_rules
+
+include struct
```

Comment:
What is the purpose of including it instead of writing this on the top-level?

Reply:
You get to avoid the `Dune_pkg` prefix. E.g. in

```
module Checksum = Dune_pkg.Checksum
module Pkg = Dune_pkg.Lock_dir.Pkg
```

the `Dune_pkg` is repeated.